packages/react-card/src/components/CardHeader/useCardHeader.ts

```diff
@@ -19,8 +19,8 @@ export const useCardHeader_unstable = (props: CardHeaderProps, ref: React.Ref<HT
       root: 'div',
       image: 'div',
       content: 'div',
-      header: React.Fragment,
-      description: React.Fragment,
+      header: 'span',
```

Comment:
I changed these to default to `span` as that is consistent with their types and allows us to apply props like `className`. Open to other approaches here if we need to keep these as fragments.

Reply:
This will break some opinionated formatting, but I have to revisit it regardless due to an issue, so all good 👌
I approved the Screener changes for Card.